lib/renderer/ipc-renderer-internal.ts

```diff
@@ -17,7 +17,7 @@ ipcRendererInternal.sendSync = function (channel, ...args) {
 ipcRendererInternal.invoke = async function<T> (channel: string, ...args: any[]) {
   const { error, result } = await ipc.invoke<T>(internal, channel, args);
   if (error) {
-    throw new Error(`Error invoking remote method '${channel}': ${error}`);
+    throw error;
```

Comment:
```suggestion
    throw new AggregateError([new Error(`Error invoking remote method '${channel}'`), error]);
```

JS has built-in primitives for this kind of thing, we should retain the "channel" context